src/main/java/step1/StringAddCalculator.java

```diff
@@ -0,0 +1,44 @@
+package step1;
+
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+    private static final String DEFAULT_DELIMITER = ",|:";
+    private static final String CUSTOM_DELIMITER = "//(.)\n(.*)";
+    private static final int ZERO = 0;
+    private static final int FIRST = 1;
+    private static final int SECOND = 2;
+    private static final SumCalculator SUM_CALCULATOR = SumCalculator.getInstance();
+
+    public static int splitAndSum(String inputString) {
+        if(isEmptyOrNullText(inputString)) {
```

Comment:
띄어쓰기 한 번 확인해주시면 좋을 것 같아요.
```suggestion
        if (isEmptyOrNullText(inputString)) {
```

Reply:
늘 if(~~) 로 사용했었는데 if (~~) 가 더 좋을까요??🤔

Reply:
넵 다음 코딩 컨벤션에서 `8. 2 공백(Blank Spaces)` 부분을 참고해주세요.
https://myeonguni.tistory.com/1596